src/main/java/lotto/LottoMain.java

```diff
@@ -0,0 +1,18 @@
+package lotto;
+
+import lotto.domain.LottoGroup;
+import lotto.domain.LottoResult;
+import lotto.view.InputView;
+import lotto.view.ResultView;
+
+public class LottoMain {
+    public static void main(String[] args) {
+        int money = InputView.getInputMoney();
+
+        LottoGroup lottoGroup = new LottoGroup(money);
+
+        ResultView.showBuyLottos(lottoGroup);
+
+        ResultView.showStatus(new LottoResult(lottoGroup, InputView.getInputWinNumber()), money);
```

Comment:
```suggestion
        Lotto winLotto = new Lotto(InputView.getInputWinNumber());
        LottoResult result = lottoGroup.match(winLotto);
        ResultView.showStatus(result, money);
```
LottoResult는 match된 결과 값을 저장하고 업데이트하는 역할만 담당하면 어떨까?
현재 코드는 LottoResult가 LottoGroup과 Lotto에 대한 협력까지 담당하다보니 너무 많은 책임을 담당하고 있는 느낌이 든다.